lib/model.js

```diff
@@ -69,6 +69,7 @@ const minimize = require('./helpers/minimize');
 const MongooseBulkSaveIncompleteError = require('./error/bulkSaveIncompleteError');
 const ObjectExpectedError = require('./error/objectExpected');
 const decorateBulkWriteResult = require('./helpers/model/decorateBulkWriteResult');
+const { ClientEncryption } = require('mongodb');
```

Comment:
Can we move this over to the driver layer? Maybe make `lib/drivers/node-mongodb-native/index.js` export `ClientEncryption`?

We currently avoid importing the Node driver directly anywhere other than `lib/drivers/node-mongodb-native' (exception being `lib/index`) because of Mongoose's browser build.

Reply:
I think I've moved the CE export to the Node driver layer.  Let me know if that isn't what you were looking for!